src/scripts/search-engines/searx.ts

```diff
@@ -0,0 +1,49 @@
+import { SEARCH_ENGINES } from '../../common/search-engines';
```

Comment:
This file is only a copy of `ecosia.ts`. Does it match the styles of the Searx instances?

I indicated some possible improvements below.

Reply:
Yes, they worked, no weird stuff seen in my tests on firefox.